index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+
+</head>
+
+<body>
+    <!-- <h1>Hello World, we are software developers</h1> -->
+    <header id="header">
+        <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo" >
+        <nav id="nav-bar">
+            <a class="nav-link" href="#premium">Features</a>
+            <a class="nav-link" href="#fast">How It Works</a>
+            <a class="nav-link" href="#quality">Pricing</a>
+        </nav>
+    </header> 
+    <main class="landingPage">
+        <h1> Handcrafted, home-made masterpieces</h1>
+
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input id="email" name="email" type="email" class="email" placeholder="Enter your email address" required>
+            <input id="submit" type="submit" class="submit" >
+    </form> 
+</main>
+
+    <section >
+    <article id="premium">
+        <i class="fa fa-3x fa-fire" style="color:orange;"></i>
+        <div class="article-bar">
+            <h1>Premium Materials</h1>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        </div>
+    </article>
+
+    <article id="fast">
+        <i class="fa fa-3x fa-truck" style="color:orange;"></i>
```

Comment:
- [ ] Please do not use inline styles. it alway a good best practices to keep your style definition in a separate CSS file you can visite [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.)